.circleci/config.yml

```diff
@@ -72,6 +93,11 @@ jobs:
           > version.json
       - run: docker build --pull -t addons-frontend .
       - run: docker images
+      - run:
+          name: Test Image
+          command: |
+            docker run --net "host" --detach addons-frontend yarn amo:dev
```

Comment:
Can't you just use the [`docker`](https://circleci.com/docs/2.0/configuration-reference/#docker) key for this instead of `machine`?

Reply:
If that would require an overhaul of other tests supported by this config then it's probably not worth the effort. I was just curious if it was easy to switch into a full docker mode.

Reply:
I don't think I can as this is after the build. I don't think it would make sense to build the image twice in different steps as that would defeat the purpose.

Reply:
From what I know, the ```docker``` tag can't build images